scripts/base/init-bare.bro

```diff
@@ -3358,6 +3358,20 @@ export {
 		share_type: count;
 	};
 
+	## The create context type information as defined for SMB2_create_request messages.
+	##
+	## For more information, see MS-SMB2:2.2.13.2
+	##
+	type SMB2::CreateContextValue: record {
+        	## The structure name indicates what information is encoded by the data payload.
+        	## Possible values are: ExtA, SecD, DHnQ, etc.
+		name        : string;
+	    	## The data payload of the type specified in the structure name.
+		data        : string;
```

Comment:
Should this data be parsed internally?  I vaguely recall running into this and avoiding it at the time because I felt like I would have needed to parse the data based on the name field in order to make it most useful to users.